sample-app/src/components/Dropdown.tsx

```diff
@@ -23,22 +29,39 @@ interface Props {
 export default function Dropdown({
   options,
   onClickOption = () => {},
+  focusedSectionIndex,
   focusedOptionIndex,
   optionIdPrefix,
   cssClasses
 }: Props): JSX.Element | null {
-  function renderOption(option: Option, index: number) {
+  function renderSection(section: { results: Option[], label?: string }, sectionIndex: number) {
```

Comment:
I might consider moving these functions to the top level of the file. I don't think it matters in this case, but react uses reference equality in a number of places, and here new `renderSection()` and `renderOption()` functions are used for each render of `Dropdown`. If we move them to the top level the functions will have stable references. AGAIN I don't think matters here because we are not passing these functions in as props anywhere or using them in as memo dependencies etc. but I also don't have a compiler in my brain